pkg/apb/svc_acct.go

```diff
@@ -86,11 +86,7 @@ func (s *ServiceAccountManager) createResources(rFilePath string, namespace stri
 	return nil
 }
 
-func (s *ServiceAccountManager) writeResourceFile(
-	handle string,
-	svcAcctM *map[string]interface{},
-	roleBindingM *map[string]interface{},
-) (string, error) {
+func (s *ServiceAccountManager) writeResourceFile(handle string, svcAcctM *map[string]interface{}, roleBindingM *map[string]interface{}) (string, error) {
```

Comment:
I really dislike long signatures on a single line like this. Anyone else feel strongly about that? @jmrodri @rthallisey? If this is a linting problem, I'd like to disable it.